resources/lang/fr/common/box_account_issues.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * Chocolatey Language Template.
+ *
+ * @template Account Issues Common Box
+ * @uri /common/box_account_issues.en.html
+ * @language en (english)
```

Comment:
As said, please change al `@language en (english)` to `@language fr (french)`
And please, change all `.en.html` to `.fr.html`